spec/integration/post_index_spec.rb

```diff
@@ -0,0 +1,71 @@
+require 'rails_helper'
+
+RSpec.describe 'posts#index', type: :feature do
+  describe 'Post' do
+    before(:each) do
+      @user = User.create(name: 'Victor',
+                          photo: 'https://ibedc.com/filesupload/imagebuild/625350c770c886.68494374.jpg',
+                          bio: 'Marketer',
+                          post_counter: 0)
+      @user.save!
+      visit root_path
+
+      @post1 = Post.create(title: 'Victors Post', text: 'My Post by Victor', comments_count: 0, likes_count: 3,
+                           author: @user)
+      @post2 = Post.create(title: 'John Post', text: 'This is my second post', comments_count: 0, likes_count: 0,
+                           author: @user)
+
+      @comment1 = Comment.create(text: 'First Comment', author: User.first,
+                                 post: @post1)
+      @comment2 = Comment.create(text: 'Second Comment', author: User.first, post: @post1)
+
+      @like = Like.create(post_id: @post1.id, author_id: @user.id)
+    end
+
+    # it "I can see the user\'s profile picture." do
+    #    visit(user_posts_path(@user.id))
+    #    expect(page).to have_css('img[src*="https://ibedc.com/filesupload/imagebuild/625350c770c886.68494374.jpg"]')
+    # end
+
+    it 'I can see the user\'s username.' do
+      visit(user_posts_path(@user.id))
+      expect(page).to have_content('Victor')
+    end
+
```

Comment:
Please add a test that asserts user can see a section for pagination if there are more posts than fit on the view on `user_posts#index`.